platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -154,62 +154,55 @@ of capabilities, platforms make it easy for their users to deliver valuable prod
-handle those operations, hiding complexity \[7\] from users. For example, users
-may often require a message queue, but shouldn't have to manage a message
-broker.
-
-Platforms are intended to make product development more efficient, so it's
-important they not be an impediment \[8\] when the most efficient path is to not
-use the platform's offerings. Platforms should enable product teams to
-provide and manage their own capabilities when necessary. For example, if a
-platform team doesn't provide a graph database and it's required for a product,
-it should be possible for that product team to provision and operate a graph
-database themselves.
+After defining what a platform is and why an organization might want to build one,
+it's relevant to identify what are the key attributes that affect the success
+of a platform.
+
+1. **Platform as a product**. A platform exists to serve the requirements of its users
+   and it should be designed and evolved based on those requirements, similar to any
+   other software products.
+1. **Common requirements**. A platform should provide the necessary capabilities to
+   support the most common use cases across stream-aligned teams, and prioritize those
+   over more specific capabilities that are only used by a single team to maximize
+   the value delivered.
+1. **User experience**. A platform should offer its capabilities through consistent
+   interfaces and focus on the user experience. Users should be able to request, manage,
+   and observe such capabilities using a combination of GUIs, APIs, command-line tools,
+   IDEs integrations, and portals, depending on their needs. For example, a platform
+   typically offers the capability of deploying an application. Developers might consume
+   such a capability via the IDE; testers might use a command-line tool, whereas a product
+   owner might use a GUI-based web portal.
+1. **Documentation and onboarding**. Documentation is a key aspect of a successful software
+   product. To be able to use a platform's offerings, users require documentation and
+   examples. A platform should be delivered with proper documentation addressing the
+   needs of its users. It should also provide tools to accelerate the onboarding of new projects
+   that can help users consume the necessary platform services in a quick and simple way.
+   For example, the platform could offer a reusable supply chain workflow for building, scanning,
+   testing, deploying, and observing a web application on Kubernetes. Such a workflow could be
+   offered with an initial project template and documentation, a bundle often described
+   as a golden path.
+1. **Self-service**. A platform should be self-serviceable. Users must be able to request and
+   receive capabilities autonomously and automatically. This property is key to allowing a platform
+   team to enable multiple stream-aligned teams at scale. The platform capabilities should be
```

Comment:
This sentence sounds a bit repetitive as is. With this edit I am trying to highlight the difference of handling current needs and scaling in the future.

```suggestion
   team to enable multiple stream-aligned teams and scale as needed. The platform capabilities should be
```